mkdocstrings_handlers/vba/_handler.py

```diff
@@ -107,6 +105,17 @@ def __init__(self, *, base_dir: Path, encoding: str, **kwargs: Any) -> None:
     **`docstring_section_style`** | `str` | The style used to render docstring sections. Options: `table`, `list`, `spacy`. | `table`
     """
 
+    def get_options(self, local_options: Mapping[str, Any]) -> Dict[str, Any]:
+        """Combine the default options with the local options.
+
+        Arguments:
+            local_options: The options provided in Markdown pages.
+
+        Returns:
+            The combined options.
+        """
+        return {**self.default_config, **local_options}
```

Comment:
I don't *think* we need to `deepcopy` here, but I've been bitten by this previously. This creates a new dict, but with references to mutable structures from both default config and local options. The only time `config` is mutated is below in `render` with `config["members_order"] = Order(config["members_order"])`, which I believe doesn't cause any issue.

We can still `deepcopy` here just to be safe. Let me know.

Reply:
Maybe the real solution is to not mutate the config at all. Otherwise yes, let's use deepcopy.